### DISCUSSION TITLE
reference/docs/bash

### DISCUSSION THREAD

**Discussion by giscus** [jaywcjlove/reference#688] [Q&A]
# reference/docs/bash

https://blog.cutterman.cn/reference/docs/bash.html

**Comment by noobalex** [comment]
# 光标移动类
| 快捷键      | 作用          |
|----------|-------------|
| Ctrl + A | 移动到命令行开头    |
| Ctrl + E | 移动到命令行末尾    |
| Alt + B  | 向左移动一个“词”   |
| Alt + F  | 向右移动一个“词”   |
| Ctrl + B | 向左移动一个字符（←） |
| Ctrl + F | 向右移动一个字符（→） |

**Reply by jaywcjlove** [reply]
https://github.com/jaywcjlove/reference/blob/91ba353e23d3fa06d9e621f9d80a61f950defa89/docs/bash.md?plain=1#L1065-L1075

@noobalex 已添加
